### ISSUE TITLE
[Programmatic Access - Logic Apps - Overview]: "Create workflow using the innovative visual designer" text present under "Get Started" tab visually appears as heading but is not defined as heading programmatically.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6101]
### Describe the issue 

User Experience:

 Visually challenged users who rely on screen readers using shortcut(H) to understand the structure of the content in the page are getting impacted, as headings are not identified when navigated via heading shortcut(H).

Note: User credentials should NOT be included in the bug.

Repro Steps:

- Open [https://ms.portal.azure.com/](<https://ms.portal.azure.com/>) and login via valid credentials. 
- Navigate to the Left Nav bar and select 'More Services', search 'Logic apps'. 
- Select any created resource. 
- Select Overview in left navigation pane. 
- Navigate all the controls on the blade like Top menu bar (Run Trigger, Refresh, Edit , Delete) , blade content , Zoom- in\out.. etc 
- Verify whether the Text present under "Get Started" tab visually appears as heading but is not defined as heading programmatically or not. 

Actual Result:

 "Create workflow using the innovative visual designer" text present under "Get Started" tab visually appears as heading but is not defined as heading programmatically.

Expected Result:

 "Create workflow using the innovative visual designer" text present under "Get Started" tab visually appears as heading should be defined as heading programmatically as

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
https://msazure.visualstudio.com/One/_workitems/edit/29381739
